test/config.js

```diff
@@ -167,3 +169,48 @@ test('valid serviceName => active', function (t) {
   t.equal(agent._conf.active, true)
   t.end()
 })
+
+var captureBodyTests = [
+  { value: 'off', errors: '[REDACTED]', transactions: '[REDACTED]' },
+  { value: 'transactions', errors: '[REDACTED]', transactions: 'test' },
+  { value: 'errors', errors: 'test', transactions: '[REDACTED]' },
+  { value: 'all', errors: 'test', transactions: 'test' }
+]
+
+captureBodyTests.forEach(captureBodyTest => {
```

Comment:
We haven't used arrow functions before in the agent. I think we should either agree to start doing it or keep to the same style just to be consistent. Though I don't have a string opinion either way

Reply:
There's also the question of maybe only using it for one line functions vs for multi line functions. But again, I don't have a strong opinion at the moment 😄

Reply:
Oh, didn't even realize I had used an arrow function there. I can just change it to a regular function.

Reply:
fixed